runners/core-java/src/test/java/org/apache/beam/runners/core/StatefulDoFnRunnerTest.java

```diff
@@ -190,7 +186,7 @@ public void testGarbageCollect() throws Exception {
 
     // advance watermark past end of WINDOW_2 + allowed lateness
     advanceInputWatermark(
-        timerInternals, WINDOW_2.maxTimestamp().plus(ALLOWED_LATENESS + 1), runner);
+        timerInternals, WINDOW_2.maxTimestamp().plus(ALLOWED_LATENESS + 2), runner);
```

Comment:
You might want to make the `2` self-documenting by something like `TimerInternalsCleanupTimer.GC_DELAY + 1` or something. Not a big deal.